packages/trader/src/AppV2/Components/ActiveSymbolsList/active-symbols-list.tsx

```diff
@@ -17,6 +21,38 @@ const ActiveSymbolsList = observer(({ isOpen, setIsOpen }: TActiveSymbolsList) =
     const [isSearching, setIsSearching] = useState(false);
     const [selectedSymbol, setSelectedSymbol] = useState(symbol);
     const [searchValue, setSearchValue] = useState('');
+    const { guideStates, setGuideState } = useGuideStates();
+    const { should_run_market_selector_guide } = guideStates;
+
+    const STEPS = [
```

Comment:
Will it make sense to move `STEPS` outside of the component? Create a file for steps config for all guides in order not to store it in multiplies places, Or you can reuse existing `steps-config.tsx` file with modifications:
```
const STEPS = {
[market_selector_guide]:[{ 
            content: <Localize i18n_default_text='Explore available markets here.' />,
            placement: 'top' as Step['placement'],
            target: '.joyride-element',
            title: <Localize i18n_default_text='Select a market' />,
            disableBeacon: true,
            offset: 0,
            spotlightPadding: 4,}],
            ...
}
```

Reply:
I think we better store it in the files where it's actually used. This way it'll be easier to modify and change it in the future.

Reply:
Agreed about having it in the same file.

Reply:
Sure, it was suggestion, I do not insist